kubernetes/charts/edge-kubernetes-crd/Chart.yaml

```diff
@@ -2,4 +2,4 @@ apiVersion: v1
 appVersion: '1.0'
 description: A Helm chart for installing CRD for Azure IoT Edge on Kubernetes
 name: edge-kubernetes-crd
-version: 0.1.6
\ No newline at end of file
+version: 0.1.7
```

Comment:
should we sync helm chart version with iotedge version?

Reply:
Let us task this for GA, since we need to figure out the right conventions for Edge appVersion vs helm chart version. And we also need to tackle user expectations about deploying multiple releases in the same namespace (which isn't possible yet I believe).